docs/getting_started/installation.md

```diff
@@ -62,3 +61,16 @@ mix deps.get
 ```
 mix phx.server
 ```
+
+## Running with Docker
```

Comment:
One more thing is missing - environment variables description, both for Docker and non-Docker runs